examples/computer_vision/README.md

```diff
@@ -0,0 +1,160 @@
+- **Interactive Web UI**: Upload images or use URLs for instant object detection testing with visual results
+- **Dataset-Model Lineage**: Full traceability linking FiftyOne datasets to ZenML model artifacts and predictions
+- **Visual Performance Analysis**: Side-by-side comparison of predictions vs ground truth in FiftyOne's interactive dashboard
+
+## 🏃 Quickstart
+
+```bash
+pip install -r requirements.txt
+zenml init
+zenml login
+```
+
+**Train with FiftyOne analysis** ([see code](pipelines/training_pipeline.py)):
+
+```bash
+# Full workflow: training + FiftyOne analysis
+python run.py --train --samples 50 --epochs 3
+
+# Fast training (skip FiftyOne analysis)
+python run.py --train --samples 50 --epochs 3 --disable-fiftyone-analysis
+```
+
+This downloads 50 COCO validation images via FiftyOne, trains a YOLOv8 nano model for 3 epochs, runs inference on the original FiftyOne dataset, and provides interactive analysis capabilities.
+
+**Deploy as a real-time service** ([see code](pipelines/inference_pipeline.py)):
+
+```bash
+zenml pipeline deploy pipelines.inference_pipeline.object_detection_inference_pipeline
+```
+
+Visit `http://localhost:8000` for the interactive UI ([see code](ui/index.html)).
+
+**Test batch inference locally**:
+
+```bash
+python run.py --predict --image https://ultralytics.com/images/bus.jpg
+```
+
+**Make predictions via API**:
+
+```bash
+curl -X POST http://localhost:8000/invoke \
+  -H "Content-Type: application/json" \
+  -d '{
+    "parameters": {
+      "image_path": "https://ultralytics.com/images/bus.jpg",
+      "confidence_threshold": 0.25
+    }
+  }'
+```
+
+**Explore with FiftyOne Dashboard**:
+
+![FiftyOne dashboard](assets/fiftyone_dashboard.png)
+
+After training, the pipeline creates persistent FiftyOne datasets linked to your ZenML model artifacts. Launch the dashboard to analyze results:
+
+```bash
+# Easy way - automatically finds datasets with predictions
+python launch_fiftyone.py
+
+# Manual way
+fiftyone app launch
+# Then in Python: fo.load_dataset('coco-2017-validation-50samples')
+```
+
+**Dataset-Artifact Connection**: Each FiftyOne dataset (named by parameters like `coco-2017-validation-50samples`) stores predictions from your latest ZenML run. Re-running pipelines overwrites predictions while preserving ground truth, so the dashboard always shows your current model's performance.
+
+In the FiftyOne dashboard you can:
+- Compare predictions vs ground truth side-by-side
+- Filter by confidence levels and object classes
+- Analyze per-class performance metrics
+- Identify false positives and negatives
+- Export problematic samples for retraining
+
+**Use the ZenML Deployment Playground**
+
+The ZenML dashboard includes a built-in playground for deployed pipelines, allowing you to test your service directly from the UI. Navigate to your deployment in the dashboard, fill in the image URL and confidence threshold, and see real-time detection results with visualizations.
```

Comment:
do you actually see the image with its results in the dashboard, too?

Reply:
Not in the zenml dashboard